config/build.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+set -euo pipefail
+
+GOCILINT_VERSION="1.31.0"
+GOCILINT_SHA256SUM="9a5d47b51442d68b718af4c7350f4406cdc087e2236a5b9ae52f37aebede6cb3"
+GOCILINT_LOCATION=https://github.com/golangci/golangci-lint/releases/download/v${GOCILINT_VERSION}/golangci-lint-${GOCILINT_VERSION}-linux-amd64.tar.gz
+
+OPERATOR_SDK_VERSION="0.16.0"
+OPERATOR_SDK_SHA256SUM="3df782f341749f7962ab0fcfedd2961c18b21ad34ff7acd194b49a152f59abcb"
+OPERATOR_SDK_LOCATION=https://github.com/operator-framework/operator-sdk/releases/download/v${OPERATOR_SDK_VERSION}/operator-sdk-v${OPERATOR_SDK_VERSION}-x86_64-linux-gnu
+
+OPM_VERSION="1.13.8"
+OPM_SHASUM="a48aa9d69b0be3439220e818edde4b36b3b9eceb2a058d86b4fdf0ca9dcd21c8"
+OPM_LOCATION=https://github.com/operator-framework/operator-registry/releases/download/v${OPM_VERSION}/linux-amd64-opm
+
+curl -L -o golangci-lint.tar.gz $GOCILINT_LOCATION
+echo ${GOCILINT_SHA256SUM} golangci-lint.tar.gz | sha256sum -c
+tar xzf golangci-lint.tar.gz golangci-lint-${GOCILINT_VERSION}-linux-amd64/golangci-lint
+mv golangci-lint-${GOCILINT_VERSION}-linux-amd64/golangci-lint /usr/local/bin
+rm -rf golangci-lint-${GOCILINT_VERSION}-linux-amd64
+rm -f golangci-lint.tar.gz
+
+curl -L -o operator-sdk $OPERATOR_SDK_LOCATION
+echo ${OPERATOR_SDK_SHA256SUM} operator-sdk | sha256sum -c
+mv operator-sdk /usr/local/bin
```

Comment:
We just got bitten by this, so I want to make sure: are we root at this point? Or is `/usr/local/bin` otherwise somehow writable?

Also, are we sure `/usr/local/bin` appears early enough in `$PATH` that we'll get the version we're installing even if there's another one installed already? And/or are we sure there's not another one installed already? (Probably the latter, or we wouldn't be here ;P )

Reply:
Yes, the docker build is done as root, so everything in writable. You make a good point about preexisting executables. I can add an extra step which delete executable on the path with the same name.